sos/plugins/lustre.py

```diff
@@ -51,4 +51,10 @@ def setup(self):
         self.get_params("quota", ["osd-*.*.quota_slave." +
                                   "{info,limit_*,acct_*}"])
 
+        # Grab emergency ring buffer dumps
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/tmp/lustre-log.*"
+            ])
+
```

Comment:
No need to wrap a single path in a list; `add_copy_spec()` accepts either a string, or an iterable of strings.